pandas/_libs/tslibs/conversion.pyx

```diff
@@ -528,7 +561,7 @@ cdef inline void localize_tso(_TSObject obj, tzinfo tz):
 
     Notes
     -----
-    Sets obj.tzinfo inplace, alters obj.dts inplace.
+    Sets obj.tzinfo inplace, alters obj.dts inplace, alters obj.value inplace.
```

Comment:
This no longer alters `obj.value`, correct?

Reply:
Thanks, reverted it.